apps/challenges/aws_utils.py

```diff
@@ -1067,6 +1068,51 @@ def get_logs_from_cloudwatch(
     return logs
 
 
+def get_tasks_status(service_name):
+    """
+    To fetch current status of workers for the given service_name
+    """
+    client = get_boto3_client("ecs", aws_keys)
+    tasks_status = {}
+    if settings.DEBUG:
+        tasks_status = {
+            "error": False,
+            "details": "Not available for development environment"
+        }
+    else:
+        try:
+            tasks_response = client.list_tasks(
```

Comment:
@burnerlee let's split this into two utils methods. First one will list tasks for a service and second method will take list of task arns as input and return response of `describe_tasks`